packages/material-ui/src/Menu/Menu.js

```diff
@@ -247,6 +382,10 @@ Menu.propTypes = {
    * `classes` prop applied to the [`Popover`](/api/popover/) element.
    */
   PopoverClasses: PropTypes.object,
+  /**
+  * @ignore
```

Comment:
should probably add some descriptions to our new props even though it appears to not be required